src/server/scripts/Northrend/VioletHold/boss_ichoron.cpp

```diff
@@ -21,54 +21,71 @@
 
 enum Spells
 {
-    SPELL_DRAINED                               = 59820,
-    SPELL_FRENZY                                = 54312,
-    SPELL_FRENZY_H                              = 59522,
-    SPELL_PROTECTIVE_BUBBLE                     = 54306,
-    SPELL_WATER_BLAST                           = 54237,
-    SPELL_WATER_BLAST_H                         = 59520,
-    SPELL_WATER_BOLT_VOLLEY                     = 54241,
-    SPELL_WATER_BOLT_VOLLEY_H                   = 59521,
-    SPELL_SPLASH                                = 59516,
-    SPELL_WATER_GLOBULE                         = 54268
+    SPELL_DRAINED                       = 59820,
```

Comment:
please try to keep codestyle and formatting changes out of the PR so the attention can be focused on real changes